src/test/resources/fixtures/deadCode/parameters-writes-only.php

```diff
@@ -2,23 +2,31 @@
 
 class Container {
     public function method1(array $in, array &$out, $log) {
-        return
+        $outc = 1;
+        $function =
             function
                 (array $inner, array &$outer)
                 use (
                     <weak_warning descr="The variable seems to be not used.">$in</weak_warning>,
-                    &$out,
+                    &<weak_warning descr="The variable seems to be not used.">$out</weak_warning>,
```

Comment:
I see and It;s totally ok for manual, just the final tests cases needs to be minimalistic.